apps/etterlatte-behandling/src/main/kotlin/Tilgangsstyring.kt

```diff
@@ -43,29 +46,56 @@ val adressebeskyttelsePlugin: RouteScopedPlugin<PluginConfiguration> = createRou
 ) {
     on(AdressebeskyttelseHook) { call ->
         val bruker = call.bruker
+
         if (bruker is SystemBruker) {
             return@on
         }
         if (call.request.uri.contains(TILGANG_ROUTE_PATH)) {
             return@on
         }
-        val behandlingId = call.parameters[BEHANDLINGSID_CALL_PARAMETER]
 
-        if (!behandlingId.isNullOrEmpty()) {
-            if (pluginConfig.behandlingIdHarAdressebeskyttelse(behandlingId)) {
-                call.respond(HttpStatusCode.NotFound)
+        if (bruker is Saksbehandler) {
```

Comment:
@oyvindsh  usikker på om vi burde denie ellers?
@EspenSchei @madsop-nav

Reply:
Hadde vore fint å få samkøyrd det til kun éin Saksbehandler og éin Systembruker, iallfall når begge brukast mest (kun?) til autentisering og autorisering så kjens det forvirrande og unødvendig å ha to.

Men om det er denne eller den andre vi bør byggje vidare på er eg litt ambivalent rundt. Synes denne verkar mest sofistikert, men den andre er mest i bruk 🤔

Reply:
Enig i det, ref spørsmålet i PRen. Samtidig brukes den andre til en del andre ting i behandling som ikke vedgår autentisering.

Reply:
Det her skaper også issues i SakService.

```
import no.nav.etterlatte.token.Saksbehandler
import no.nav.etterlatte.Saksbehandler as SaksbehandlerBruker
```

Ikke ryddig - men - de har forskjellige info på seg og begge trengs forskjellige steder.

Reply:
@chrissearle Jeg finner ikke de i SakService nå?

Reply:
@sebassonav Prøv igjen - kom nettopp inn

Reply:
Aha men jeg flyttet `harTilgangTilPerson` fra SakService så det går nok fint.

Reply:
Synes uansett at det virker noe feil med oppsett med to Saksbehandler klasser.

Reply:
Viss begge skal leve vidare (ref "Samtidig brukes den andre til en del andre ting i behandling som ikke vedgår autentisering."), bør vi sikte oss inn på å separere dei ordentleg og vera tydelege på kva som er ansvarsområda for kvar av dei, og truleg også døpe om litt.

Reply:
@chrissearle @madsop-nav https://jira.adeo.no/browse/EY-2013